nss-day-cohort-60/rare-client-tenacious-tators
# Jes subscribed posts

Closes #7


# Description of Proposed Changes

Adds subscribed posts to the homepage, if user has not subscribed to any other users a paragraph should show on the page saying to do so.

---


# Steps to Test

1. **You must be running the server branch that accompanies this code.** If you don't have that branch fetch jes-view-subscribed-posts from the server repo. See instructions on that[ PR](https://github.com/nss-day-cohort-60/rare-python-server-tyrannical-tators/pull/19) to make sure you have subsciptions in your DB
2. fetch the [jes-subscribed-posts](https://github.com/nss-day-cohort-60/rare-client-tenacious-tators/compare/main...jes-subscribed-posts?expand=1) and switch to it
3. login as user with id of 1 on your local SQL DB (if you followed server side INSERT statements this user has all the subscriptions)
4. navigate to the "/" page by clicking the logo in the navbar
5. Verify that you are seeing the correct posts that the user is subscribed to.
6.  Logout and login as a user that isn't subscribed
7. navigate to the "/" page by clicking the logo in the navbar
8. Verify that "Subscribe to authors to curate your personal homepage" is displayed.
